.github/workflows/accept-baselines-fix-lints.yaml

```diff
@@ -16,11 +16,10 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
-    permissions:
-      contents: write
-
     steps:
       - uses: actions/checkout@v3
```

Comment:
do we want this to be v4 also?

Reply:
Surprisingly, this doesn't do a full fetch, so I don't need to bump it.

I wanted to bump all of the versions but I'd need to update every file's version specifiers which felt noisy. Probably, we should enable dependabot but just for GHA. But either of those I'd rather in a different PR.